### DISCUSSION TITLE
Can I serialize ClientResponse objects?

### DISCUSSION THREAD

**Discussion by cloasdata** [aio-libs/aiohttp#6802] [Q&A]
Hello Folks,

is it easily possible to serialize objects of ClientResponse?

Thanks a lot!
seimen

**Comment by Dreamsorcerer** [comment]
Surely not, it may have a live connection open.

Maybe explain what you are actually trying to do, and we can suggest a better alternative.

**Reply by cloasdata** [reply]
Id just like to pass the response when closed to a worker through a queue (threaded or multiprocess).
The worker may not need all the attributes but some of them are useful. Of course I could write an adapter or something similar. Thats why I asked for easiness :)

So at the end a sync .text() could also be useful. Not sure if the protected _body is what Im looking for.

**Reply by Dreamsorcerer** [reply]
I'd suggest just passing the explicit data you need (e.g `{"status": resp.status, "text": await resp.text()}`).
